gc/verbose/handler_standard/VerboseHandlerOutputStandard.cpp

```diff
@@ -403,9 +404,10 @@ MM_VerboseHandlerOutputStandard::handleScavengeEndNoLock(J9HookInterface** hook,
 	MM_VerboseWriterChain* writer = manager->getWriterChain();
 	MM_ScavengerStats *scavengerStats = &extensions->incrementScavengerStats;
```

Comment:
`scavengerStats` reference can be removed here if nothing is making use of it now